templates/OperationResponseNoBodySnippet.stencil

```diff
@@ -1,8 +1,20 @@
 switch result {
-case .success:
-    DispatchQueue.main.async {
-        completionHandler(.success(()), httpResponse)
+case let .success(data):
+    {% if op.exceptions.count > 0 %}
+    {% include "OperationResponseData.stencil" %}
+    {% endif %}
```

Comment:
So we still render the data section on a no-body scenario if there could be exceptions?

Reply:
the if logic 'op.exceptions.count > 0' is to make it  render the data section only when there are exceptions.